.github/workflows/ci.yml

```
@@ -616,10 +616,15 @@ jobs:
         uses: ./.github/actions/run-maven
         with:
           parameters: -f build-tools/pom.xml install
+      - name: Build optimize dependencies
+        uses: ./.github/actions/run-maven
+        with:
+          parameters: -pl optimize/backend -am install -Dquickly -Dskip.fe.build -PskipFrontendBuild -Dskip.docker
```

```suggestion
          parameters: -pl optimize/backend -am install -Dquickly
```
I think this can be simplified and have the same effect with just the `quickly` flag